src/main/java/Ui.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Scanner;
+
+/** Describes the wrapper class for handling UI. */
+public class Ui {
+    public final String INVALID_ARGUMENT = "Sorry, you have supplied invalid arguments.";
+    public final String INVALID_COMMAND = "Sorry, I don't understand the command.";
+    public final String ADD_TASK = "I have added the task to the task list!";
+    public final String SHOW_TASKS = "Here are your tasks!";
+    public final String COMPLETE_TASK = "I have marked that task as done!";
+    public final String DELETE_TASK = "I have deleted that task from the task list!";
+    public final String FIND_TASKS = "Finding tasks that match your search keyword...";
```

Comment:
I like how you named messages for different scenarios, which improves the readability of the code.